openlibrary/macros/SearchResultsWork.html

```diff
@@ -34,12 +38,26 @@ <h3 class="booktitle">
             <a itemprop="author" href="$(a.get('url') or a.get('key') or a.author.get('url') or a.author.key)" class="results">$(a.name or a.author.name)</a>$(',' if not loop.last else '')
       </span>
       <span class="resultPublisher">
-        $if doc.edition_count:
-          $ungettext('1 edition', '%(count)d editions', doc.edition_count, count=doc.edition_count)
-          $if doc.first_publish_year:
-              $_('- first published in %(year)s', year=doc.first_publish_year)
+        $if doc.get('first_publish_year'):
+          <div>
+            $_('First published in %(year)s', year=doc.first_publish_year)
+          </div>
+        $if doc.get('edition_count'):
+          <a href="$(book_url)?edition=$(ocaid)#editions-list">$ungettext('1 edition', '%(count)d editions', doc.edition_count, count=doc.edition_count)</a>
+          $if doc.get('languages'):
+            <span class="languages">
+              $_('in') <a title="$(', '.join(doc.languages))">$(len(doc.languages)) $_('languages')</a>
+            </span>
+          $if doc.get('ia'):
+            &mdash; $(len(doc.get('ia'))) $_('previewable')
+            $if len(doc.get('ia')) > 1:
+              <div class="preview-covers">
+                $for x, i in enumerate(doc.get('ia')[1:10]):
+                  <a href="$(book_url)?edition=$(i)">
+                    <img width="30" height="45" src="//archive.org/download/$(i)/page/cover_w60_h60.jpg">
```

Comment:
:( Open a new issue for these.

Reply:
https://github.com/internetarchive/openlibrary/issues/3556

Reply:
The solution (using `openlibrary_edition` from the availability API) also included in tech requirements and solved by this branch.